.github/workflows/ci-storybook.yml

```
@@ -66,23 +66,11 @@ jobs:
 
   test-storybook:
     runs-on: ubuntu-latest
-    timeout-minutes: 10
-
-    needs: [build-storybook]
+    timeout-minutes: 15
 
     steps:
       - uses: actions/checkout@v5
       - uses: ./.github/actions/pnpm-install
```

actionlint (1.7.12)

[error] missing input "bryntum_access_token" which is required by action "pnpm-install" defined at "./.github/actions/pnpm-install". all required inputs are "bryntum_access_token"

(action)